template/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,106 @@
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+require 'dotenv'
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Push a new beta build to TestFlight"
+  lane :beta do
+    build_app(workspace: "your_workspace", scheme: "") # enter your beta scheme
+    upload_to_testflight(
+        skip_waiting_for_build_processing: true
+    )
+  end
+end
+
+platform :ios do
+  desc "Push a new beta build to TestFlight"
+  lane :betaDev do
+    build_app(workspace: "your_workspace", scheme: "") # enter your dev scheme
+    upload_to_testflight(
+        skip_waiting_for_build_processing: true
+    )
+  end
+
+  desc "Push a new beta build local to TestFlight"
+  lane :buildLocal do
+    Dotenv.load('../../.env.local')
+    build_app(workspace: "your_workspace", scheme: "") # enter your local scheme
+    upload_to_testflight(
+        skip_waiting_for_build_processing: true,
+        changelog: "#{ENV["APPCENTER_RELEASE_NOTE"]}"
+    )
+  end
+
+  desc "Push a new beta build staging to TestFlight"
+  lane :buildStaging do
+    Dotenv.load('../../.env.staging')
+    build_app(workspace: "your_workspace", scheme: "") # enter your staging scheme
+    upload_to_testflight(
+        skip_waiting_for_build_processing: true,
+        changelog: "#{ENV["APPCENTER_RELEASE_NOTE"]}"
+    )
+  end
+
+  desc "Push a new beta build Preprod to TestFlight"
+  lane :buildPreprod do
+    Dotenv.load('../../.env.preprod')
+    build_app(workspace: "your_workspace", scheme: "") # enter your preprod scheme
+    upload_to_testflight(
+        skip_waiting_for_build_processing: true,
+        changelog: "#{ENV["APPCENTER_RELEASE_NOTE"]}"
+    )
+  end
+
+  desc "Push a new beta build Prod to TestFlight"
+  lane :buildProd do
+    Dotenv.load('../../.env.prod')
+    build_app(workspace: "your_workspace", scheme: "") # enter your prod scheme
+    upload_to_testflight(
+        skip_waiting_for_build_processing: true,
+        changelog: "#{ENV["APPCENTER_RELEASE_NOTE"]}"
+    )
+  end
+end
+
+platform :ios do
+  desc "Code push local"
+  lane :codePushLocal do 
+    Dotenv.load('../../.env.local')
+    sh "cd .. && cd .. && code-push release-react <ownerName>/MyApp ios -d Local"
+  end
+
+  desc "Code push staging"
+  lane :codePushStaging do 
+    Dotenv.load('../../.env.staging')
+    sh "cd .. && cd .. && code-push release-react <ownerName>/MyApp ios"
+  end
+
+  desc "Code push preprod"
+  lane :codePushPreprod do 
+    Dotenv.load('../../.env.preprod')
+    sh "cd .. && cd .. && code-push release-react <ownerName>/MyApp ios -d PreProd"
+  end
+
+  desc "Code push production"
+  lane :codePushProd do 
+    Dotenv.load('../../.env.prod')
+    sh "cd .. && cd .. && code-push release-react <ownerName>/MyApp ios -d Production"
```

Comment:
Consider adding variable to `<ownerName>/MyApp` (including other similar code above)